### PR TITLE
Adds speakers API to CFS route

### LINKED ISSUE #795: Add speaker API to public/cfs route
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

Add speaker details via modal.
Follow the same implementation as on eventyay.

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Adds speakers API to CFS
- Adds new-speakers route

Fixes #795

### CODE REVIEW

**Comment by geekyd** [comment]
@niranjan Can you please tell me what route should I use for editing the speaker info? Should I reuse the current route or should I follow the same implementation as eventyay? Please help

**Comment by niranjan94** [comment]
Deployment link: https://pr-803-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@niranjan94 Can you please help?

**Comment by geekyd** [comment]
@fossasia/open-event-frontend Need help guys. Should we reuse the new-speaker route or have a separate routes for create & edit ?

**Comment by sumedh123** [comment]
@geekyd I think can should use the same ```new-speaker``` route.

**Comment by niranjan94** [comment]
Deployment link: https://pr-803-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@sumedh123 @CosmicCoder96 Current approach uses the same route for edit & ceration. Please review.

File: app/controllers/public/cfs/new-speaker.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  actions: {
+    save() {
+      this.set('isLoading', true);
+      this.get('model.speaker').save()
+        .then(() => {
+          this.transitionToRoute('public.cfs.index');
```
Comment (CosmicCoder96) [inline app/controllers/public/cfs/new-speaker.js]: @geekyd Please give a success notification too.

**Comment by niranjan94** [comment]
Deployment link: https://pr-803-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-803-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-803-open-event-frontend-fossasia.surge.sh

**Comment by geekyd** [comment]
@CosmicCoder96 please review again. Thanks

**Pull request merged**